Behavioral.Automation.DemoBindings/AutomationConfig.json

```diff
@@ -2,7 +2,7 @@
   "BASE_URL": "http://localhost:4200/",
   "TEST_EMAIL": "",
   "TEST_PASSWORD": "",
-  "BROWSER_PARAMS": "--window-size=1920,1080",
+  "BROWSER_PARAMS": "--window-size=1920,1080 --allowed-ips",
```

Comment:
Does it make sense to run tests in a headless mode?

Reply:
this setting allows tests to access external services, moved this parameter to action variables, should not affect running in headless mode